components/rewards/RewardsHome.js

```diff
@@ -1,71 +1,68 @@
 import React from 'react';
-import { Platform, StyleSheet, Text, View } from 'react-native';
-import { Body } from '../BaseComponents';
+import { ScrollView, Text, View } from 'react-native';
+import { Body, Overline } from '../BaseComponents';
+import { ProgressBar, Colors } from 'react-native-paper';
+import RewardsCard from '../../components/rewards/RewardsCard';
+import { ContentContainer, WaluigiContainer } from '../../styled/rewards';
+
 /**
  * @prop
  * */
 
+function createList(N) {
+  let foo = [];
+
+  for (var i = 1; i <= N; i++) {
+    foo.push(i);
+  }
+  return foo;
+}
+
 function RewardsHome({ transactions, user }) {
   return (
     <View>
-      <View style={styles.getStartedContainer}>
-        <Text> {`Welcome, ${user.name}`}</Text>
-        <Body>{user.points} total points </Body>
-        <Text style={styles.getStartedText}>
-          {' '}
-          {`${1000 -
-            (parseInt(user.points) % 1000)} points to your next reward`}
-        </Text>
-      </View>
-
-      <View style={styles.tabBarInfoContainer}>
-        <Text style={styles.tabBarInfoText}>Your transaction history:</Text>
-        {transactions.splice(0, 3).map(transaction => {
-          return (
-            <View key={transaction.id}>
-              <Body style={styles.codeHighlightText}>
-                Date: {transaction.Date}
-              </Body>
-              <Body style={styles.codeHighlightText}>
-                Points Redeemed: {transaction['Points Rewarded']}
-              </Body>
-            </View>
-          );
-        })}
-      </View>
+      <ScrollView style={{ marginTop: 45 }}>
+        <WaluigiContainer>
+          <Overline>REWARD PROGRESS</Overline>
+          <Text style={{ fontSize: 10 }}> </Text>
+          <Text
+            style={{
+              fontSize: 30
+            }}>
+            {parseInt(user.points) % 1000} / 1000
+          </Text>
```

Comment:
Tommy created a base component style for this in a recent PR called Big Title (so update this when that gets merged).